aws/main.tf

```diff
@@ -34,3 +34,24 @@ resource "aws_lambda_function" "my_hello_world" {
   }
 }
 
+resource "aws_instance" "new_web_app" {
+  ami           = "ami-005e54dee72cc1d00"
+
+  instance_type = "m5.4xlarge"
+
+  tags = {
+    Environment = "production"
+    Service     = "web-app"
+  }
+
+  root_block_device {
+    volume_size = 100
+  }
```

Comment:
```suggestion
    volume_type = "gp3"
  }
```


🔴 **EBS - consider upgrading gp2 volumes to gp3**

gp3 volumes are the latest generation of general-purpose SSD-based EBS volumes that enable you to provision performance independent of storage capacity, while providing up to 20% lower price per GB than existing gp2 volumes. With gp3 volumes, you can scale IOPS (input/output operations per second) and throughput without needing to provision additional block storage capacity. This means you only pay for the storage you need.